content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -67,14 +67,16 @@ the resource that should be available on the node.
 
 On Linux, the kubelet uses the following eviction signals:
 
-| Eviction Signal      | Description                                                                           |
-|----------------------|---------------------------------------------------------------------------------------|
-| `memory.available`   | `memory.available` := `node.status.capacity[memory]` - `node.stats.memory.workingSet` |
-| `nodefs.available`   | `nodefs.available` := `node.stats.fs.available`                                       |
-| `nodefs.inodesFree`  | `nodefs.inodesFree` := `node.stats.fs.inodesFree`                                     |
-| `imagefs.available`  | `imagefs.available` := `node.stats.runtime.imagefs.available`                         |
-| `imagefs.inodesFree` | `imagefs.inodesFree` := `node.stats.runtime.imagefs.inodesFree`                       |
-| `pid.available`      | `pid.available` := `node.stats.rlimit.maxpid` - `node.stats.rlimit.curproc`           |
+| Eviction Signal          | Description                                                                           |
+|--------------------------|---------------------------------------------------------------------------------------|
+| `memory.available`       | `memory.available` := `node.status.capacity[memory]` - `node.stats.memory.workingSet` |
+| `nodefs.available`       | `nodefs.available` := `node.stats.fs.available`                                       |
+| `nodefs.inodesFree`      | `nodefs.inodesFree` := `node.stats.fs.inodesFree`                                     |
+| `imagefs.available`      | `imagefs.available` := `node.stats.runtime.imagefs.available`                         |
+| `imagefs.inodesFree`     | `imagefs.inodesFree` := `node.stats.runtime.imagefs.inodesFree`                       |
+| `containerfs.available`  | `containerfs.available` := `node.stats.runtime.containerfs.available`                 |
+| `containerfs.inodesFree` | `containerfs.inodesFree` := `node.stats.runtime.containerfs.inodesFree`               |
```

Comment:
should we mention these are part of the split image feature gate? are they available if the gate is not on (I forget)

Reply:
These metrics will be available if memory serves me right, albeit I will double-check the code...

However, I didn't mention the feature gate anywhere, as we also have plans to graduate our KEP to "Beta" with this release, hopefully. So, with the flag turned on as the new default, these would be made available.

That said, I will mention the feature gate so that there is some notion of it, especially in a case should people want to switch it off, perhaps.

Reply:
yeah I think mentioning it is a good idea

Reply:
Done.